gjs doesn't set a SONAME by default which leads to bad paths being baked
into the binaries.

XXX file upstream bug

--- a/Makefile.am	2017-10-31 12:41:28.207749943 -0700
+++ b/Makefile.am	2017-10-31 12:41:53.733672392 -0700
@@ -70,6 +70,7 @@
 	$(AM_LDFLAGS)			\
 	-export-symbols-regex "^[^_]"	\
 	-version-info 0:0:0		\
+        -Wl,-soname=libgjs.so.0		\
 	$(NO_UNDEFINED_FLAG)		\
 	$(NULL)
 libgjs_la_LIBADD = 		\
